### PR TITLE
[BUILD] Use MAVEN_ARGS for passing Maven CLI arguments

### PR DESCRIPTION
**Pull request by bowenliang123**
# :mag: Description
## Issue References 🔗

This pull request fixes #

## Describe Your Solution 🔧

- inspired by comments from #6664 : https://github.com/apache/kyuubi/pull/6664#issuecomment-2344351119

## Types of changes :bookmark:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Related Unit Tests

---

# Checklist 📝

- [x] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -106,10 +106,10 @@ jobs:
       - name: Build and test Kyuubi and Spark with maven w/o linters
         run: |
           if [[ "${{ matrix.java }}" == "8" && "${{ matrix.spark }}" == "3.5" && "${{ matrix.spark-archive }}" == "" ]]; then
-            MVN_OPT="${MVN_OPT} -Pcodecov -Dmaven.javadoc.skip=false -Dmaven.scaladoc.skip=false"
+            MAVEN_ARGS="${MAVEN_ARGS} -Pcodecov -Dmaven.javadoc.skip=false -Dmaven.scaladoc.skip=false"
```
Comment (pan3793) [inline .github/workflows/master.yml:109]: in shell, we must use `export MAVEN_ARGS=...` to declare the environment varaiable

Comment (pan3793) [inline .github/workflows/master.yml:109]:
```suggestion
            export MAVEN_ARGS="${MAVEN_ARGS} -Pcodecov -Dmaven.javadoc.skip=false -Dmaven.scaladoc.skip=false"
```

Comment (bowenliang123) [reply .github/workflows/master.yml]: OK, thanks.

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
looks good to go, any concerns?

**Comment by bowenliang123** [comment]
LGTM.

**Comment by bowenliang123** [comment]
Thanks, merged to master (1.10.0).

**Pull request closed without merging**